### PR TITLE
Add Service FRs

### PR DESCRIPTION
**Pull request by joyxiao99**
Close https://github.com/arkinmodi/project-sayyara/issues/70

### CODE REVIEW

File: docs/SRS/SRS.tex [hunk 1]
```diff
@@ -338,6 +339,62 @@ \subsubsection{Authentication}
...
+		      \item Viewpoint: Vehicle Owner
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Owner
+		            \begin{enumerate}
+			            \item The system shall allow the user to enter the name of the service
+			            \item The system shall allow the user to enter a description for the service
+			            \item The system shall allow the user to enter the estimated time for the service
+			            \item The system shall allow the user to enter the parts used for the service including quantity,
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:355]: Nabeel mentioned types of parts used for service: e.g., OEM, used, new

File: docs/SRS/SRS.tex [hunk 2]
```diff
@@ -338,6 +339,62 @@ \subsubsection{Authentication}
...
+			                  condition, build, and cost per part
+			            \item The system shall allow the user to enter the total price for the service
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:357]: I'm guessing some fields are optional here... do we want to add "optionally enter" or something similar for optional service fields

File: docs/SRS/SRS.tex [hunk 3]
```diff
@@ -338,6 +339,62 @@ \subsubsection{Authentication}
...
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Employee
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+	      \end{enumerate}
+
+	\item The user wants to search for shop services
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:365]: nit: auto repair or maintenance services

File: docs/SRS/SRS.tex [hunk 4]
```diff
@@ -338,6 +339,62 @@ \subsubsection{Authentication}
...
+	      \begin{enumerate}[VP\arabic*.]
+		      \item Viewpoint: Vehicle Owner
+		            \begin{enumerate}
+			            \item The system shall display the service details
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:369]: The system should take input for the search

File: docs/SRS/SRS.tex (hunk 2 above)
Comment (joyxiao99) [reply docs/SRS/SRS.tex:357]: I thought it was just showing that we are letting the user enter it.. doesn't mean they HAVE to

**Review by arkinmodi — APPROVED** [review]

**Pull request merged**